plugins/destination/postgresql/client/write.go

```diff
@@ -31,52 +32,37 @@ func (c *Client) Write(ctx context.Context, table string, data map[string]interf
 	return nil
 }
 
-func insert(table string, data map[string]interface{}) (string, []interface{}) {
-	var sb strings.Builder
+func insert(table string, data map[string]interface{}) (string, []any) {
```

Comment:
I wouldn't switch `[]interface{}` to `[]any` unless we are going to make the change everywhere

Reply:
for now, `any` is just an alias of `interface{}`, so in this case I propose to gradually convert to `any` (as it's a bit cleaner)